### PR TITLE
Jupyter lab support

### PR DESCRIPTION
**Pull request by chlubnamarek**
Percent separators support added
Databricks notebooks deployment updated
Ignoring jupyter notebooks checkpoints folders for deployment

### CODE REVIEW

File: src/dbxdeploy/_config/config.yaml [hunk 1]
```diff
@@ -334,7 +341,17 @@ services:
 
   dbxdeploy.notebook.converter.CellsExtractor:
 
-  dbxdeploy.notebook.converter.DatabricksNotebookConverter:
+#  dbxdeploy.notebook.converter.DatabricksNotebookConverter:
```
Comment (kutny) [inline src/dbxdeploy/_config/config.yaml:344]: Remove this pls

**Review by kutny — CHANGES_REQUESTED** [review]

File: src/dbxdeploy/_config/config.yaml [hunk 2]
```diff
@@ -13,6 +13,7 @@ parameters:
         patterns: ['**/*.py']
         consumers_patterns: ['**/*.consumer.py']
         master_package_notebook_path: '%dbxdeploy.source.notebooks.base_dir%/app/install_master_package.py'
+        cell_separator: 'percent' # command or percent
```
Comment (kutny) [inline src/dbxdeploy/_config/config.yaml:16]: Just note: I would choose "command" as default value for now as this is still a bit experimental feature and we want to make it backward compatible.

File: src/dbxdeploy/notebook/loader.py [hunk 3]
```diff
@@ -2,5 +2,8 @@
 
 
 def load_notebook(notebook_path: Path) -> str:
```
Comment (kutny) [inline src/dbxdeploy/notebook/loader.py:4]: Why haven't you changed NotebooksLocator class instead?

**Review by kutny — CHANGES_REQUESTED** [review]

**Review by guderkar — APPROVED** [review]
ok

**Pull request merged**